datacapture/src/main/java/com/google/android/fhir/datacapture/validation/ValidationUtil.kt

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.android.fhir.datacapture.validation
+
+import ca.uhn.fhir.context.FhirContext
+import ca.uhn.fhir.context.FhirVersionEnum
+import ca.uhn.fhir.context.support.DefaultProfileValidationSupport
+import com.google.android.fhir.datacapture.CQF_CALCULATED_EXPRESSION_URL
+import org.hl7.fhir.r4.hapi.ctx.HapiWorkerContext
+import org.hl7.fhir.r4.model.Expression
+import org.hl7.fhir.r4.model.Type
+import org.hl7.fhir.r4.utils.FHIRPathEngine
+
+object ValidationUtil {
+
+  internal val fhirPathEngine: FHIRPathEngine =
```

Comment:
Make this `private` as this is only used in this file.
```suggestion
  private val fhirPathEngine: FHIRPathEngine =
```

Reply:
actually valueOrCalculatedValue is an extension that is out of scope of ValidationUtil that is why it was made internal so extension can access it. I have refactored ValidationUtil to use it as a file now.